<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QBitMPlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="topLevelHorizontalLayout" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="playlistVerticalLayout">
        <item>
         <widget class="QLabel" name="playlistLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Playlist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="playlistWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="playlistControlsHorizontalLayout">
          <item>
           <widget class="QPushButton" name="openPlaylistButton">
            <property name="text">
             <string>Open</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closePlayListButton">
            <property name="text">
             <string>Close</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::ListRemove"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savePlaylistButton">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removePlaylistButton">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::EditDelete"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="playingAreaVerticalLayout" stretch="0,0,0,0,0">
        <item>
         <layout class="QHBoxLayout" name="playingHorizontalLayout">
          <item>
           <widget class="QLabel" name="playingLabel">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Playing:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playingEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openFilesButton">
            <property name="text">
             <string>Open Files</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="durationHorizontalLayout">
          <item>
           <widget class="QSlider" name="seekMusicSlider">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="durationLabel">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="controlsHorizontalLayout" stretch="1,1,1,0">
          <item>
           <layout class="QVBoxLayout" name="playStopVerticalLayout">
            <item>
             <widget class="QPushButton" name="playButton">
              <property name="text">
               <string>Play</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopButton">
              <property name="text">
               <string>Stop</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="ChangeMusicVerticalLayout">
            <item>
             <widget class="QPushButton" name="previousButton">
              <property name="text">
               <string>Previous</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::MediaSeekBackward"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextButton">
              <property name="text">
               <string>Next</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::MediaSeekForward"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="seekVerticalLayout">
            <item>
             <widget class="QPushButton" name="seekBackwardButton">
              <property name="text">
               <string>Seek Backward</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="seekForwardButton">
              <property name="text">
               <string>Seek Forward</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="autoRepeatButton">
              <property name="text">
               <string>Auto Repeat</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::SystemReboot"/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="volumeHorizontalLayout">
              <item>
               <widget class="QLabel" name="volumeLabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Volume:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="volumeSlider">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="volumeIconButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="musicControlsHorizontalLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFiles"/>
    <addaction name="actionOpen_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionOpenPlaylist"/>
    <addaction name="actionClosePlaylist"/>
    <addaction name="actionSavePlaylist"/>
    <addaction name="actionRemovePlaylist"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>Audio</string>
    </property>
    <widget class="QMenu" name="menuDevices">
     <property name="title">
      <string>Devices</string>
     </property>
    </widget>
    <addaction name="menuDevices"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAudio"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFiles">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>&amp;Open Files</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowClose"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionOpenPlaylist">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
   </property>
   <property name="text">
    <string>Open &amp;Playlist</string>
   </property>
  </action>
  <action name="actionClosePlaylist">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove"/>
   </property>
   <property name="text">
    <string>&amp;Close Playlist</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Sa&amp;ve Playlist</string>
   </property>
  </action>
  <action name="actionRemovePlaylist">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>&amp;Remove Playlist</string>
   </property>
  </action>
  <action name="actionOpen_Directory">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
